scripts/pre_commit_linter.py

```diff
@@ -44,19 +44,34 @@
 
 __author__ = "Barnabas Makonda(barnabasmakonda@gmail.com)"
 
+
 import argparse
 import os
 import subprocess
 import sys
 import time
 
-
 _PARSER = argparse.ArgumentParser()
 _PARSER.add_argument(
     '--path',
     help='path to the directory with files to be linted',
     action='store')
 
+PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+
+
+_PYLINT_PATH = os.path.join(
+    PARENT_DIR, 'oppia_tools', 'pylint-1.5.2')
+if not os.path.exists(_PYLINT_PATH):
+    print ''
+    print 'ERROR    Please run start.sh first to install pylint '
+    print '         and its dependencies.'
+    sys.exit(1)
+
+sys.path.append(_PYLINT_PATH)
+from pylint import epylint as lint
```

Comment:
Could not do ` import pylint` because it could not find the `epylint` module, but removed the renaming aspect, so it is now `from pylint import epylint` with the corresponding changes in the file.